stable/xray/templates/NOTES.txt

```diff
@@ -1,6 +1,6 @@
 Congratulations! JFrog Xray services are deployed!
 
-{{- if eq .Values.xray.masterKey "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF" }}
+{{- if and (eq .Values.xray.masterKey "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF") .Values.xray.masterKeySecretName }}
```

Comment:
@lindhe  I think it should be  `(not .Values.xray.masterKeySecretName)` right ?

Reply:
Certainly. My fault for throwing together this minutes before leaving for the day. Feel free to change it if you want, or I'll do it when I find time. Likely in a few hours.

Reply:
Thanks, Please update the PR when you get a chance :)

Reply:
I think I was able to do it from my phone. Unless I screwed up, I feel very cool.